src/table/cells/handle-cell-focus.js

```diff
@@ -1,67 +1,75 @@
-export const updateFocus = ({ rowElements = {}, cellCoord = [], shouldFocus = true, providedCell = undefined }) => {
+export const findCellWithTabStop = (rootElement) => rootElement.querySelector("td[tabindex='0'], th[tabindex='0']");
+
+export const updateFocus = ({ focusType, rowElements = [], cellCoord = [], providedCell = undefined }) => {
   const cell = providedCell || rowElements[cellCoord[0]]?.getElementsByClassName('sn-table-cell')[cellCoord[1]];
-  if (cell) {
-    if (shouldFocus) {
+  if (!cell) return;
+
+  switch (focusType) {
```

Comment:
The move of this focus is to get the blue border I guess? Is tabindex needed as well for this?

Reply:
yes. We want to be able to tab to the pagination stuff in the footer, then tab back